docs/runtime_attributes.md

```diff
@@ -0,0 +1,19 @@
+# Key Runtime Attributes
+
+This doc is intended to capture the key runtime attributes for Node.js.  
+These attributes will be used along with the use-cases in order to
+guide what we develop/run as part of the community benchmarking
+efforts.
+
+# Attributes
+
+The following attributes have been identified so far:
+
+* startup/stop time
+* memmory footprint at startup
```

Comment:
In this case what I had in mind was the resident set after the runtime has started and is ready to run the application. Also interesting would be the peak between 0 and that time.  My first guess is that since the startup is relatively quick that they could be the same for Node.

When you say allocation throughput do you mean in terms of startup or as a separate attribute ?